modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -378,22 +386,22 @@ Do not use these parameters with the `-I` or `S` parameters to configure REST AP
 `--amc-host`
 | `_amc-host_`
 
-|Specifies the service location URL of your local instance of Runtime Manager, such as `+https://10.0.0.1:8080/hybrid/v1+`.
+|Specifies the service location URL of your local instance of Runtime Manager, such as `+https://10.0.0.1:443/hybrid/v1+`.
```

Comment:
I notice that your examples are introduced differently:

- This one with a comma and "such as"
- The next with a colon and "for example,"
- The next two with a comma and "for example,"

What I would do is end with a period each sentence that begins with "Specifies". Then, use this construction: "Example: `+https...whatever`", with no period at the end.

Whether you do that or not, each example should be introduced in the same way.

Reply:
This is different than the SG guidance, but made them consistent.